demos/selenium-test-app/README.md

```diff
@@ -5,6 +5,8 @@ This app is used by the Internet Identity Selenium tests. It contains additional
 ## Getting Started
```

Comment:
Why is it called `selenium-test-app`?

For what I can see it's a normal dapp, isn't it?

Reply:
How exactly do we tamper with the behavior? Also that's a big-ish design decision (for this test app), can we record that somewhere in the README?